py/float.go

```diff
@@ -398,14 +398,14 @@ func (a Float) M__ge__(other Object) (Object, error) {
 func init() {
 	FloatType.Dict["is_integer"] = MustNewMethod("is_integer", func(self Object) (Object, error) {
 		if a, ok := convertToFloat(self); ok {
-			f, err := FloatAsFloat64(a)
-			if err != nil {
-				return nil, err
+			f, _ := FloatAsFloat64(a)
```

Comment:
Please return nil, err when FloatAsFloat64 return error

Reply:
Ok, I'll change that.